pootle/apps/pootle_app/management/commands/__init__.py

```diff
@@ -82,6 +83,20 @@ def do_translation_project(self, tp, **options):
                              self.name, store.pootle_path)
                 self.handle_store(store, **options)
 
+    def check_projects(self, project_codes):
+        for project_code in project_codes:
+            try:
+                Project.objects.get(code=project_code)
+            except Project.DoesNotExist as e:
+                raise CommandError(e)
+
+    def check_languages(self, language_codes):
+        for language_code in language_codes:
```

Comment:
im thinking it would be more efficient to do something like
```
try:
     assert len(Language.objects.filter(code__in=language_codes)) == len(language_codes)
except AssertionError:
     raise CommandError("Fooed!")
```

same above